<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryActionMenu</class>
 <widget class="QDialog" name="HistoryActionMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>511</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="5" column="0">
      <widget class="QPushButton" name="btStop">
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/stop.png</normaloff>
         <disabledoff>:/icons/stop-disbl.png</disabledoff>
         <disabledon>:/icons/stop-disbl.png</disabledon>:/icons/stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="btAddContact">
       <property name="text">
        <string>Add contact</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/add-contact.png</normaloff>
         <disabledoff>:/icons/add-contact-disbl.png</disabledoff>
         <disabledon>:/icons/add-contact-disbl.png</disabledon>:/icons/add-contact.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btCall">
       <property name="text">
        <string>Call</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/call.png</normaloff>
         <disabledoff>:/icons/call-disbl.png</disabledoff>
         <disabledon>:/icons/call-disbl.png</disabledon>:/icons/call.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="btDeleteAll">
       <property name="text">
        <string>Delete all</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/delete-all.png</normaloff>
         <disabledoff>:/icons/delete-all-disbl.png</disabledoff>
         <disabledon>:/icons/delete-all-disbl.png</disabledon>:/icons/delete-all.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btPlay">
       <property name="text">
        <string>Play</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/play.png</normaloff>
         <disabledoff>:/icons/play-disbl.png</disabledoff>
         <disabledon>:/icons/play-disbl.png</disabledon>:/icons/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="btDelete">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/delete.png</normaloff>
         <disabledoff>:/icons/delete-disbl.png</disabledoff>
         <disabledon>:/icons/delete-disbl.png</disabledon>:/icons/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="btHistorySettings">
       <property name="text">
        <string>Settings</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/settings-w.png</normaloff>
         <disabledoff>:/icons/settings-w-disbl.png</disabledoff>
         <disabledon>:/icons/settings-w-disbl.png</disabledon>:/icons/settings-w.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="btDetails">
       <property name="text">
        <string>Details</string>
       </property>
       <property name="icon">
        <iconset resource="taki.qrc">
         <normaloff>:/icons/details.png</normaloff>
         <disabledoff>:/icons/details-disbl.png</disabledoff>
         <disabledon>:/icons/details-disbl.png</disabledon>:/icons/details.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="taki.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btDetails</sender>
   <signal>released()</signal>
   <receiver>HistoryActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAddContact</sender>
   <signal>released()</signal>
   <receiver>HistoryActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDelete</sender>
   <signal>released()</signal>
   <receiver>HistoryActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDeleteAll</sender>
   <signal>released()</signal>
   <receiver>HistoryActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCall</sender>
   <signal>released()</signal>
   <receiver>HistoryActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btStop</sender>
   <signal>released()</signal>
   <receiver>HistoryActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btHistorySettings</sender>
   <signal>released()</signal>
   <receiver>HistoryActionMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClicked()</slot>
 </slots>
</ui>
